doc/cascadia/profiles.schema.json

```diff
@@ -54,6 +54,12 @@
         }
       ]
     },
+    "BellSound": {
+      "default": "#",
+      "description": "Controls the sound the Terminal plays when the application emits a BEL character. If the path is invalid no sound will be played. This property also accepts an array of sounds and the Terminal will pick one at random.",
+      "pattern": "",
+      "type": "string",
```

Comment:
I suggest taking a look at how we handle `backgroundImage`...
https://github.com/microsoft/terminal/blob/1aa717a60d9e7cded96b83fda98c88be24adbb94/doc/cascadia/profiles.schema.json#L132-L151

We should be able to accept null, a string, or an array of strings.

Reply:
`BellStyle` is another good example to look at. It should show you how to make an array:
https://github.com/microsoft/terminal/blob/1aa717a60d9e7cded96b83fda98c88be24adbb94/doc/cascadia/profiles.schema.json#L34-L44

Reply:
I believe I captured the values correctly.